### ISSUE TITLE
Agent icons stay on map when hired to second base

### ISSUE THREAD

**Issue by Kurtsley** [OpenApoc/OpenApoc#1274]
When hiring or transferring an agent to the starting base, the agent icon on the map goes away as soon as they arrive. This icon seems to appear when you have an agent in the city awaiting pickup. When a second base is purchased, this icon stays after the agent arrives. Not a big deal, but it bothers me. 

The icon:
![agent](https://github.com/OpenApoc/OpenApoc/assets/73447098/01e337a5-ced5-4019-b15f-6559d70f64a6)

**Comment by FilmBoy84** [comment]
Agreed, the icon should have a yellow line drawn to it's destination location and move with the agent as they are transported through the city as per OG
